### PR TITLE
Fix nil G in constructor

### PR DESCRIPTION
**Pull request by patrickxb**
r? @maxtaco

### CODE REVIEW

File: go/service/pgp.go [hunk 1]
```diff
@@ -162,7 +162,7 @@ func (h *PGPHandler) PGPImport(_ context.Context, arg keybase1.PGPImportArg) err
 		LogUI:     h.getLogUI(arg.SessionID),
 		SessionID: arg.SessionID,
 	}
-	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, nil)
+	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, h.G())
```
Comment (maxtaco) [inline go/service/pgp.go]: I thought contextified still worked in this case to grab the global G if the passed `g` is `nil`

Comment (maxtaco) [inline go/service/pgp.go]: OK, I looked into it.  I think there are two bugs here.  One, as you found, and the other that `StringWarning#Warn` should take a `libkb.Contextified` which is resilient to this type of `nil` bug.

Comment (patrickxb) [inline go/service/pgp.go]: Well, libkb.Warning doesn't use contextified:

https://github.com/keybase/client/blob/master/go/libkb/warning.go

and there's this:

https://github.com/keybase/client/blob/master/go/engine/pgp_import_key.go#L58

**Comment by maxtaco** [comment]
👍

File: go/service/pgp.go (hunk 1 above)
Comment (patrickxb) [inline go/service/pgp.go]: The final commit on warning.go says "don't contextualize", so there must have been a reason...

https://github.com/keybase/client/commit/a813b5d14642cfcfe957c95d422abd5798800125#diff-626dc7bce8452b387b5dbfc56efd261aL38

Comment (maxtaco) [inline go/service/pgp.go]: OK, sounds good as is

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 7c1cac24a312...e3bbf01f66dd introduced some visual changes:
 * Changed: **unlock_folders-Success** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466620049790-7c1cac24a312-e3bbf01f66dd/unlock_folders-Success-7c1cac24a312.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466620049790-7c1cac24a312-e3bbf01f66dd/unlock_folders-Success-e3bbf01f66dd.png)  
   unlock_folders-Success comparison
 * Changed: **Username_Email_Login-Name_Error** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466620049790-7c1cac24a312-e3bbf01f66dd/Username_Email_Login-Name_Error-7c1cac24a312.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466620049790-7c1cac24a312-e3bbf01f66dd/Username_Email_Login-Name_Error-e3bbf01f66dd.png)  
   Username_Email_Login-Name_Error comparison

**Comment by chromakode** [comment]
The visual diffs reported here are spurious. Sorry about that!

**Pull request merged**